CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 CHANGELOG
 =========
 
+## 0.27.4
+
+### Improvements
+
+- Added secret support for all fields in DeploymentSettings [#419](https://github.com/pulumi/pulumi-pulumiservice/issues/419)
```

Comment:
Heh, it seems like I'm just noticing this, but the CHANGELOG should link to the PRs that fix the issues instead of the issues themselves. Let's fix it for this one and going forward.

Reply:
Updated to have PR links instead, but now that I think of it, how can you link to a PR that doesn't exist until the change? 🤔 

Like, I make an edit, add a line to CHANGELOG, but where do I link? I guess can predict the number based on latest, but yeah

Reply:
I usually do a new commit for the changelog once I open the PR and know what the number is